examples/apps/echo/client/echo_client.py

```diff
@@ -29,274 +29,339 @@
 import utility.signature as signature
 import utility.hex_utils as hex_utils
 from error_code.error_status import SignatureStatus
+from work_order_receipt.work_order_receipt_request import WorkOrderReceiptRequest
 
 # Remove duplicate loggers
 for handler in logging.root.handlers[:]:
     logging.root.removeHandler(handler)
 logger = logging.getLogger(__name__)
-TCFHOME = os.environ.get("TCF_HOME", "../../")
-
-def ParseCommandLine(args) :
-
-	global worker_obj
-	global worker_id
-	global message
-	global config
-	global off_chain
-	global requester_signature
-	global input_data_hash
-
-	parser = argparse.ArgumentParser()
-	use_service = parser.add_mutually_exclusive_group()
-	parser.add_argument("-c", "--config", 
-		help="the config file containing the Ethereum contract information", 
-		type=str)
-	use_service.add_argument("-r", "--registry-list", 
-		help="the Ethereum address of the registry list", 
-		type=str)
-	use_service.add_argument("-s", "--service-uri", 
-		help="skip URI lookup and send to specified URI", 
-		type=str)
-	use_service.add_argument("-o", "--off-chain", 
-		help="skip URI lookup and use the registry in the config file", 
-		action="store_true")
-	parser.add_argument("-w", "--worker-id", 
-		help="skip worker lookup and retrieve specified worker", 
-		type=str)
-	parser.add_argument("-m", "--message", 
-		help='text message to be included in the JSON request payload', 
-		type=str)
-	parser.add_argument("-rs", "--requester-signature",
-		help="Enable requester signature for work order requests",
-		action="store_true")
-	parser.add_argument("-dh", "--data-hash",
-		help="Enable input data hash for work order requests",
-		action="store_true")
-
-	options = parser.parse_args(args)
-
-	if options.config:
-		conf_files = [options.config]
-	else:
-		conf_files = [ TCFHOME + \
-			"/examples/common/python/connectors/tcf_connector.toml" ]
-	confpaths = [ "." ]
-	try :
-		config = pconfig.parse_configuration_files(conf_files, confpaths)
-		json.dumps(config)
-	except pconfig.ConfigurationException as e :
-		logger.error(str(e))
-		sys.exit(-1)
-
-	global direct_jrpc
-	direct_jrpc = DirectJsonRpcApiConnector(conf_files[0])
-
-	# Whether or not to connect to the registry list on the blockchain
-	off_chain = False
-
-	if options.registry_list:
-		config["ethereum"]["direct_registry_contract_address"] = \
-			options.registry_list
-
-	if options.service_uri:
-		config["tcf"]["json_rpc_uri"] = options.service_uri
-		off_chain = True
-
-	if options.off_chain:
-		off_chain = True
-
-	requester_signature = options.requester_signature
-	input_data_hash = options.data_hash
-	worker_id = options.worker_id
-	message = options.message
-	if options.message is None or options.message == "":
-		message = "Test Message"
-
-	# Initializing Worker Object
-	worker_obj = worker.SGXWorkerDetails()
+TCFHOME = os.environ.get("TCF_HOME", "../../../../")
+
+def ParseCommandLine(args):
```

Comment:
This file has too many changes which is unrelated to Work order receipt implementation. Can you please move the formatting related changes to separate PR

Reply:
Formatting changes are related to remove tab and space mix usages. I don't want to remove it to separate PR.